Android/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(MCServer)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
```

Comment:
I've only just seen this after your reference in #3161. But what I meant was why cant you do `CMAKE_GENERATOR ${CMAKE_GENERATOR}`?